Source/Atomic/Input/Input.h

```diff
@@ -290,6 +292,12 @@ class ATOMIC_API Input : public Object
     /// Return whether application window is minimized.
     bool IsMinimized() const;
 
+    /// Binds UIButton element to the given button
+    void BindButton(UIButton* touchButton, int button);
+
+    void FakeButtonDown(int button);
+    void FakeButtonUp(int button);

```

Comment:
How about "Simulate" instead of "Fake", so SimulateButtonDown and SimulateButtonUp?